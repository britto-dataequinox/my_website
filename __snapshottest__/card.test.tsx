import React from 'react';
import renderer from 'react-test-renderer';
import ResponsiveCard from '../components/Card';

test('card snap', () => {
  const tree = renderer .create(<ResponsiveCard title="Title" content="Content" width="300px" margin="10px" />).toJSON();
  expect(tree).toMatchSnapshot();
});
