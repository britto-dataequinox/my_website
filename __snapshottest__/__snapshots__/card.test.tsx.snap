// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card snap 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1bxgd24-MuiPaper-root-MuiCard-root"
>
  <div
    className="MuiCardContent-root css-46bh2p-MuiCardContent-root"
  >
    <div
      className="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom css-h93ljk-MuiTypography-root"
    >
      Title
    </div>
    <div
      className="MuiTypography-root MuiTypography-body2 css-r40f8v-MuiTypography-root"
    >
      Content
    </div>
  </div>
</div>
`;
